app/helpers/Enable_Disable_email.py

```diff
@@ -0,0 +1,8 @@
+from flask import render_template
+from .email import send_email
+from .token import generate_token
+
+
+def SendEmail(email , name , status , subject , sender , template , app):
```

Comment:
Please **delete** this function and the file its in. We already have an existing `send_email` function in `app/helpers/email.py`, you dont need to duplicate it.